django/db/backends/mysql/schema.py

```diff
@@ -103,6 +103,15 @@ def _column_default_sql(self, field):
             return "(%s)"
         return super()._column_default_sql(field)
 
+    def _wrapper_sql_for_default(self, default):
+        # The only database function MySQL supports before 8.0.13 is
+        # CURRENT_TIMESTAMP. It uses the same formatting as literals.
+        if not self.connection.features.supports_expression_defaults:
+            return "%s"
+
+        # MySQL after 8.0.13 uses the same formatting as other databases.
+        return super()._wrapper_sql_for_default(default)
```

Comment:
Is this [still](https://github.com/django/django/pull/16092/files#r1160593472) required?

Reply:
I think this really comes down to what Django's support policy is for MySQL before 8.0.13. If we don't support it we can delete this. Otherwise we need to keep it or we'll get database errors for `Now`.

Reply:
:thinking: As far as I'm aware, we now raise `fields.E011` on MySQL < 8.0.13 when `db_default` is used so `_wrapper_sql_for_default()` should not be reached on MySQL < 8.0.13. Am I right?